src/main/java/com/google/cloud/tools/ide/login/GoogleLoginState.java

```diff
@@ -352,7 +235,7 @@ public boolean logOut() {
    *         if the user chose not to log out
    */
   public boolean logOut(boolean showPrompt) {
```

Comment:
Does it make sense to have a logout for a roster?  Add `@deprecated`?  Or delegate to a new method `clear()`?